lib/klaviyo/apis/identify.rb

```diff
@@ -0,0 +1,23 @@
+module Klaviyo
+  class Identify < Client
+    def self.identify(kwargs = {})
```

Comment:
is there a reason we have someone pass in a hash rather than specifically specify arguments?